src/main/java/net/sacredlabyrinth/phaed/simpleclans/chat/handlers/SpigotChatHandler.java

```diff
@@ -47,6 +47,24 @@ public void run() {
         }.runTask(plugin);
     }
 
+    private String stripColorsAndFormatsPerPermission(ClanPlayer sender, String message) {
+        if (!plugin.getPermissionsManager().has(sender.toPlayer(), "simpleclans.member.chat.color")) {
+            message = stripColors(message);
+        }
+        if (!plugin.getPermissionsManager().has(sender.toPlayer(), "simpleclans.member.chat.format")) {
+            message = stripFormats(message);
+        }
+        return message;
+    }
```

Comment:
stripColors method already exists in [ChatUtils](https://github.com/RoinujNosde/SimpleClans/blob/master/src/main/java/net/sacredlabyrinth/phaed/simpleclans/utils/ChatUtils.java#L57).

stripFormat should be renamed to don't confuse with a format from config and moved to ChatUtils as well. 
 
Also can be simplified to
```java
boolean hasColors = !permManager.has();
boolean hasFormat = !permManager.has();
return hasColors ? stripColors() : hasFormat ? stripFormat() : message;
```
(Considered as already in ChatUtils)
(Less readable probably :thinking:)

Reply:
> stripColors method already exists in [ChatUtils](https://github.com/RoinujNosde/SimpleClans/blob/master/src/main/java/net/sacredlabyrinth/phaed/simpleclans/utils/ChatUtils.java#L57).

They are different, ChatUtils#stripColors removes colors _and_ formats. For this feature I can't use that method.

Reply:
> ChatUtils#stripColors

Nice name of method then. 👺

Reply:
Indeed. 😄